src/main/java/core/basesyntax/IFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+interface IFigure {
+    void draw();
+}
```

Comment:
Interface names should not start with 'I'. It's a Java convention to name interfaces without the 'I' prefix, which is more common in languages like C#. Consider renaming your interface to something more descriptive without the prefix, such as 'Figure'.